resources/migrations/001_update_migrations.yaml

```diff
@@ -10509,6 +10509,106 @@ databaseChangeLog:
                   type: ${timestamp_type}
                   remarks: The timestamp at which a user was deactivated
 
+
+  - changeSet:
+      id: v53.2025-02-03T08:34:51
```

Comment:
is this feature going to land on 53? if not we should change this to 54

Reply:
Thanks, updated!